contents/docs/reading-data.mdx

```diff
@@ -168,7 +168,7 @@ You can filter a query with `where()`:
 z.query.issue.where('priority', '=', 'high');
 ```
 
-The first parameter is always a column name from the table being queried. Intellisense will offer available options (sourced from your [Zero Schema](/docs/zero-schema)).
+The first parameter is always a column name from the table being queried. TypeScript completion will offer available options (sourced from your [Zero Schema](/docs/zero-schema)).
```

Comment:
Good edit. I think many people don't know the term Intellisense, and it's probably specific to vscode.